src/data-models/mdm-data-element.resource.ts

```diff
@@ -204,9 +205,26 @@ export class MdmDataElementResource extends MdmResource {
     otherDataModelId: Uuid,
     otherDataClassId: Uuid,
     dataElementId: Uuid,
-    options?: RequestSettings
+    options?: RequestSettings,
   ) {
     const url = `${this.apiEndpoint}/dataModels/${dataModelId}/dataClasses/${dataClassId}/dataElements/${otherDataModelId}/${otherDataClassId}/${dataElementId}`;
     return this.simplePost(url, {}, options);
   }
+
+  /**
+   * `HTTP POST` - Same as above but takes a payload object instead of individual parameters, allows renaming of copy.
+   *
+   * @param dataElementId The unique identifier of the data element to copy.
+   * @param copyDataElementPayload The payload object containing the source and target data model and data class identifiers.
+   *
+   * `200 OK` - will return a {@link DataElementDetailResponse} containing the new copy of a {@link DataElementDetail} object.
+   */
+  copy(
+    dataElementId: Uuid,
+    copyDataElementPayload: CopyDataElementPayload,
```

Comment:
Same with this method:

- [ ] Move `target...` and `source...` properties of the payload into parameters
- [ ] `CopyDataElementPayload` should only contain `copyLabel` and `copyPermissions`